config.json

```diff
@@ -1,7 +1,7 @@
 {
 	"admin_server": {
 		"listen_url": "127.0.0.1:3333",
-		"use_tls": true,
+		"use_tls": false,
```

Comment:
We'll want to reset this to `true` before merging.

Reply:
Also just a reminder that we'll want to reset this to `true` before merging.

Reply:
fixed